chart/templates/services/api/_container.tpl

```diff
@@ -43,6 +44,8 @@
     value: {{ .Values.api.uvicornNumWorkers | quote }}
   - name: API_UVICORN_PORT
     value: {{ .Values.api.uvicornPort | quote }}
+  volumeMounts:
+  {{ include "volumeMountCachedAssetsRO" . | nindent 2 }}
```

Comment:
RO in volumeMountCachedAssetsRO was for read-only. Here, we need to have access in writing mode -> RW

Reply:
```suggestion
  {{ include "volumeMountCachedAssetsRW" . | nindent 2 }}
```